torch/_higher_order_ops/invoke_subgraph.py

```diff
@@ -97,6 +97,58 @@ def _invoke_subgraph_placeholder_wrapper(func, args):
     return func(*args, **kwargs)
 
 
+def export_cache(name):
+    """
+    export_cache is a hackier version of @mark_compiled_region.
```

Comment:
Why not just reuse the mark_compiled_region API? Having two different APIs is confusing

Reply:
Would it be ok for mark_compile_region to take a name?